odbc_fdw.c

```diff
@@ -526,12 +542,14 @@ sql_data_type(
 		case SQL_BIGINT :
 			appendStringInfo(sql_type, "bigint");
 			break;
+		/*
```

Comment:
Yes, the translation is correct (SQL_BINARY to bit(...) and SQL_VARBINARY to varbit(...)) but we reading thos values is complicated, so we don't support it yet. So we leave the sql_type blank which now makes the column to be ignored.